changelog/core-memory-__delete.dd

```diff
@@ -0,0 +1,35 @@
+`core.memory.__delete` has been added
+
+$(REF __delete, core, memory) allows easy migration from the deprecated `delete`.
+`__delete` behaves exactly like $(LINK2 $(ROOT_DIR)spec/expression.html#delete_expressions, `delete`):
```

Comment:
One of the procedures we need to perform to deprecate and eventually remove `delete` is to remove all references to it in the spec (see https://github.com/dlang/dlang.org/pull/1941).  I suggest removing the link to the spec.